CMakeLists.txt

```diff
@@ -102,7 +102,6 @@ include_directories(${NMODL_PROJECT_SOURCE_DIR}
                     ${NMODL_PROJECT_SOURCE_DIR}/src
                     ${PROJECT_BINARY_DIR}/src
                     ${NMODL_PROJECT_SOURCE_DIR}/ext
-                    ${NMODL_PROJECT_SOURCE_DIR}/ext/fmt/include
```

Comment:
It would probably be worthwhile to clean up the other includes on this list - like I said elsewhere, they are basically like a global variable (with all those problems).